src/components/widgets/stats/PrinterStatsCard.vue

```diff
@@ -105,6 +118,18 @@ export default class PrinterStatsCard extends Vue {
   get supportsHistoryComponent () {
     return this.$store.getters['server/componentSupport']('history')
   }
+
+  handleResetStats () {
+    this.$confirm(
+      this.$tc('app.history.msg.confirm_stats'),
+      { title: this.$tc('app.general.label.confirm'), color: 'card-heading', icon: '$error' }
+    )
+      .then(res => {
```

Comment:
My personal preference goes to use `async..await` when possible (it should work fine here) but I have no problem with promise-style code!

Reply:
Fair enough! 🙂